deployments/configurator.nix

```diff
@@ -11,6 +11,10 @@ with (import <nixpkgs> {}).lib;
     # for NixOS configs however, we need to make a distinction between defaults and and user-supplied parameters.
     imports = [ <module/configurator.nix> ];
 
+    nixpkgs.overlays = [
+      (import <overlay/monitoring-exporters.nix>)
+    ];
+
```

Comment:
i have since added that to `modules/monitoring-exporters.nix`, so if your using the existing monitoring stuff, you dont need to add the overlay yourself